src/components/canrawview/canrawview_p.h

```diff
@@ -44,6 +50,31 @@ class CanRawViewPrivate : public QWidget {
 
     ~CanRawViewPrivate() {}
 
+    void saveSettings(QJsonObject& json)
+    {
+        QJsonObject jObjects;
+        QJsonArray viewModelsArray;
+        /*
+        QString fileName("viewSave.cds");
+        QFile saveFile(fileName);
+
+        if (saveFile.open(QIODevice::WriteOnly) == false) {
+            cds_debug("Problem with open the file {0} to write confiruration", fileName.toStdString());
+            return;
+        }
+        */
+        writeColumnsOrder(jObjects);
+        jObjects["Sorting"] = prevIndex;
+        jObjects["Scrolling"] = (ui->freezeBox->isChecked() == true) ? 1 : 0;
+        writeViewModel(viewModelsArray);
+        jObjects["Models"] = viewModelsArray;
+        json[q_ptr->windowTitle().toStdString().c_str()] = jObjects;
```

Comment:
Assert added. Generally window has some title, it is require during window creation, nevertheless I added the protection against empty string.